core/src/main/java/com/aem/demo/core/utils/ResolverUtil.java

```diff
@@ -0,0 +1,25 @@
+package com.aem.demo.core.utils;
+
+import org.apache.sling.api.resource.LoginException;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ResourceResolverFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class ResolverUtil {
+
+    private ResolverUtil() {
```

Comment:
It's great you didn't forget the private constructor! (thumb_up)
Just keep in mind, a constructor is a method, and all instance methods go behind static methods. Therefore, it's more proper to have this constructor at the very bottom of the class. Because all other methods are static